WeightProducer/python/MCSampleValues.py

```diff
@@ -1784,10 +1784,13 @@ def __init__(self, extra_dicts=None):
         if extra_dicts is not None:
             self.__values_dict.update(extra_dicts)
 
+    def add_dict(self,extra_dicts):
```

Comment:
Why not change the `extra_dicts` argument in the `__init__` function to take a list of dictionaries? While I'm not opposed to an `add_dict` function, I think this could make the interface in MCSample.py a little clearner.

Reply:
The functionality for adding multiple extra custom dictionaries has now been moved to the constructor